tests/dopamine/utils/lock_test.py

```diff
@@ -0,0 +1,72 @@
+# coding=utf-8
+# Copyright 2018 The Dopamine Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for lock_decorator.py."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from dopamine.utils import lock
+from tensorflow import test
+
+
+class _MockLock(object):
```

Comment:
I think this can be made easier with the "mock" library.  go/python/testing/mocking is a good starting point.  If you'd like someone to look at this together with, I'd do so enthusiastically.

Reply:
:+1: